bessctl/module_tests/worker_split.py

```diff
@@ -62,6 +62,52 @@ def test_worker_split(self):
 
             bess.reset_all()
 
+    def test_worker_split_mask_mode(self):
+        NUM_WORKERS = 6
```

Comment:
Maybe note that this must be even? (seems kind of obvious though)

Reply:
No enforced with `multiprocessing.cpu_count() & 0xFE`